### PR TITLE
Hide graph script and make revisit default

### PR DESCRIPTION
**Pull request by gerazo**

### CODE REVIEW

File: tools/xtu-build-new/readme.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 * You need to create a compilation database json of your build process as this is the source of information used by all XTU scripts here.
 
 ## Process
 These are the steps of XTU analysation:
 
 1. `xtu-build` script uses your compilation database and extracts all necessary information from files compiled.
   It puts all its generated data into a folder (.xtu by default).
-2. `xtu-build-graph` script creates an optimal plan on doing the XTU analysis from the data generated by step 1.
-3. `xtu-analyse` script uses all previously generated data and executes the analysation.
+2. `xtu-analyse` script uses all previously generated data and executes the analysation.
```
Comment (dkrupp) [inline tools/xtu-build-new/readme.md]: the script is called xtu-analyze.py

File: tools/xtu-build-new/readme.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
   It needs the clang binary and scan-build-py's analyze-cc in order to do that.
   The output is put into a folder (.xtu-out by default)
   where both the analysation reports and the called commands' generated output is stored.
 
 ## Usage example
 0. You have generated your compilation database into build.json and you are in your projects build directory
 1. `xtu-build -b build.json -v --clang-path <path-to-folder-of-clang-binary>`
-2. `xtu-build-graph -b build.json -o graph.json`
-3. `xtu-analyse -b build.json -g graph.json -v --clang-path <path-to-folder-of-clang-binary> --analyze-cc-path <path-to-folder-of-analyze-cc-script>`
+2. `xtu-analyse -b build.json -v --clang-path <path-to-folder-of-clang-binary> --analyze-cc-path <path-to-folder-of-analyze-cc-script>`
```
Comment (dkrupp) [inline tools/xtu-build-new/readme.md]: script is called xtu-analyze.py

File: tools/xtu-build-new/readme.md [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
-2. `xtu-build-graph` script creates an optimal plan on doing the XTU analysis from the data generated by step 1.
-3. `xtu-analyse` script uses all previously generated data and executes the analysation.
+2. `xtu-analyse` script uses all previously generated data and executes the analysation.
   It needs the clang binary and scan-build-py's analyze-cc in order to do that.
   The output is put into a folder (.xtu-out by default)
   where both the analysation reports and the called commands' generated output is stored.
 
 ## Usage example
 0. You have generated your compilation database into build.json and you are in your projects build directory
 1. `xtu-build -b build.json -v --clang-path <path-to-folder-of-clang-binary>`
```
Comment (dkrupp) [inline tools/xtu-build-new/readme.md]: xtu-build.py

File: tools/xtu-build-new/readme.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
 
 ## Requirements
-* You need to have your project to be analysed set up for building
+* You need to have your to be analysed project correctly set up for the build.
 * You need to create a compilation database json of your build process as this is the source of information used by all XTU scripts here.
 
 ## Process
 These are the steps of XTU analysation:
 
 1. `xtu-build` script uses your compilation database and extracts all necessary information from files compiled.
```
Comment (dkrupp) [inline tools/xtu-build-new/readme.md]: xtu-build.py

**Comment by gerazo** [comment]
Correct scriptnames are used in readme now.

**Pull request merged**